src/main/java/org/folio/circulation/domain/ItemStatus.java

```diff
@@ -32,7 +39,7 @@ public static ItemStatus from(String value, String date) {
   }
 
   private final String value;
-
+  // FIXME: Enum constants are singletons, date must not be associated with it
```

Comment:
What do you propose as the fix for this? Have a separate class which contains the status itself and the date?

Reply:
Yes, I think so.

Reply:
What might that class be called?

Reply:
Maybe we can rename the enum to `ItemSatusName` and use the existing name for the class with enum and date.

Reply:
I like that idea